tests/test_endpoints.py

```diff
@@ -9,6 +9,13 @@ def test_heartbeat(server: str, context: BrowserContext):
     assert resp.status == 200
 
 
+def test_config(server_config, to_review_enabled):
+    assert server_config["project_name"]
```

Comment:
do we want to add 

```py
assert server_config["to_review_enabled"] == "dev" not in server_config["project_name"].lower():
```

Reply:
Yeah that seems reasonable. I think I'll need some parenthesis though